recipes/colour-demosaicing/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "colour-demosaicing" %}
+{% set version = "0.2.1" %}
```

Comment:
Latest version is 0.2.4, if you need to start with 0.2.1 it is fine but if not, upgrading will save you some PRs later.

Reply:
sounds good!